daemonize.py

```diff
@@ -118,4 +119,8 @@ def start(self):
 
         self.logger.warn("Starting daemon.")
 
-        self.action()
+        try:
+            self.action()
+        except Exception, e:
+            for line in traceback.format_exc(e):
```

Comment:
As far as I can remember `traceback.format_exc` returns string instead of list of string, so this code will log every single character of traceback in a separate line.